Makefile

```diff
@@ -80,10 +89,31 @@ all: build
 
 include docker/Makefile
 
+.PHONY: go-check
+go-check:
+	$(eval GO_MISSING := $(shell hash go > /dev/null 2>&1 || echo 1))
```

Comment:
This should check the go version 😄

Reply:
Ideally yes, but `go version` output isn't so easy to parse.

Reply:
Check added, it's `sh` compatible.